### DISCUSSION TITLE
Weird Disallow entries in robots.txt

### DISCUSSION THREAD

**Discussion by molangning** [community/community#68029] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, everyone!

I was looking through Github for `robots.txt` and `security.txt` files when I noticed something interesting, which is that some users are disallowed, namely account-login, Explodingstuff and ekansa.

While I am in no position to judge the reason behind disallowing these users, disallowing these users is quite a weird thing to do.

**So why has Github disallowed these users?**

Thanks in advance!

**Comment by Red-K0** [comment, marked as answer]
ekansa was [an SEO issue.](https://twitter.com/ekansa/status/1137052076062650368)
explodingstuff hosts WannaCry, the disallow prevents the site from being flagged as malware.
account-login according to github staff was just because that user was being placed as the top result for "github login", rather than the actual login page.

**Reply by molangning** [reply]
Thank you!

**Comment by QuickWrite** [comment]
The `opencontext` repositories by the user **ekansa** in the `robots.txt` were blocked because of:
> Ill-conceived experiment to use GitHub for public version control of some of [@OpenContext](https://x.com/OpenContext)'s legacy XML. Web crawlers went from Open Context to a gazillion XML documents in GitHub. Evidently they didn't like it.
~ https://twitter.com/ekansa/status/1137052076062650368
